belgif-rest-problem-java-ee/src/main/java/io/github/belgif/rest/problem/ee/internal/ConstraintViolationUtil.java

```diff
@@ -45,56 +48,81 @@ private ConstraintViolationUtil() {
     }
 
     public static InputValidationIssue convertToInputValidationIssue(ConstraintViolation<?> violation) {
-        LinkedList<Node> propertyPath = new LinkedList<>();
-        Iterator<Node> pathIterator = violation.getPropertyPath().iterator();
         MethodNode methodNode = null;
+        List<Node> propertyPath = new ArrayList<>();
+        List<String> propertyName = new ArrayList<>();
+        Iterator<Node> pathIterator = violation.getPropertyPath().iterator();
         while (pathIterator.hasNext()) {
             Node p = pathIterator.next();
+            propertyPath.add(p);
             if (p.getKind() == ElementKind.METHOD) {
                 methodNode = p.as(MethodNode.class);
             }
             if ((p.getKind() != ElementKind.METHOD && p.getKind() != ElementKind.PARAMETER)
                     || !pathIterator.hasNext()) {
                 // skip leading METHOD or PARAMETER path segments,
                 // e.g. "sector" instead of "createPartner.partner.sector"
-                propertyPath.add(p);
+                propertyName.add(p.toString());
```

Comment:
I'm not sure if this is correct for the BeanParam scenario. The `JaxRsParameterNameProvider` only uses the annotation value for constructor and method parameters.

Reply:
I made the changes to retrieve the input name from the JAX-RS param annotations.

Reply:
We should probably add FormParam, MatrixParam and CookieParam to the SOURCE_MAPPING.

* FormParam can map to "in": "body", see https://spec.openapis.org/oas/v3.0#example-basic-multipart-form
* MatrixParam can map to "in": "path", see https://spec.openapis.org/oas/v3.0#style-values
* CookieParam should map to "in": "cookie", see https://spec.openapis.org/oas/v3.0#parameter-locations
  -> but then it should be added to the enum in problem-v1.yaml: https://github.com/belgif/openapi-problem/blob/af2d348540a7e4aa58f19a100dcd1b89fd303b74/src/main/openapi/problem/v1/problem-v1.yaml#L141
What do we do in the meantime? Leave "in" empty?